src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9026,6 +9026,10 @@ public sealed partial class IsByRefLikeAttribute : System.Attribute
     public static partial class IsConst
     {
     }
+    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+    public static class IsExternalInit
```

Comment:
```suggestion
    public static partial class IsExternalInit
```
to match the auto-generated format